src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/PricingType.php

```diff
@@ -95,6 +108,27 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_tag_name' => 'h2',
                 'label_help_box' => $this->trans('Set specific prices for customers meeting certain conditions.', 'Admin.Catalog.Help'),
             ])
+            ->add('show_catalog_price_rules', IconButtonType::class, [
+                'icon' => 'visibility',
+                'label' => $this->trans('Show catalog price rules', 'Admin.Catalog.Feature'),
+            ])
+            ->add('hide_catalog_price_rules', IconButtonType::class, [
+                'icon' => 'visibility_off',
+                'attr' => [
+                    'class' => 'd-none btn-default',
+                ],
+                'label' => $this->trans('Hide catalog price rules', 'Admin.Catalog.Feature'),
+            ])
+            ->add('catalog_price_rules', CatalogPriceRulesType::class, [
+                'label' => $this->trans('Catalog price rules', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h2',
+                'attr' => [
+                  'data-catalog-price-url' => $catalogPriceRuleLink,
+                ],
+                'row_attr' => [
+                    'class' => 'd-none',
+                ],
+            ])
```

Comment:
```suggestion
```

To confirm with @MatShir and @TristanLDD but I don't think we need to be able to toggle the list once it's hidden, and regarding the link towards the catalog price rules you can use the `external_link` option

Besides you only need the link to the list for this redirection, not the update one with placeholder, and you can't use `admin_catalog_price_rules_index` for the external link either since it's not officially migrated.

The `$catalogPriceRuleLink` which includes a placeholder is useful for the list so it should be attached to the `show_catalog_price_rules` button instead